src/style.css

```diff
@@ -1,3 +1,76 @@
-body {
+body,
+header,
+a,
+ul {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`. Also, don't use `*` selector as it impacts performance. Set styles only for elements that require them. Zeroing out your margins, paddings or other styles with '*' is still inefficient for browser.